pipeline/alignments/generate-space-alignment.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+##
+# Generates space tokenized alignments to work with OpusTrainer Tags
+#
+
+set -x
+set -euo pipefail
+
+echo "###### Generating alignments"
+[[ -z "${BIN}" ]] && echo "BIN is empty"
+[[ -z "${SRC}" ]] && echo "SRC is empty"
+[[ -z "${TRG}" ]] && echo "TRG is empty"
+
+corpus_prefix=$1
+output_prefix=$2
+
+COMPRESSION_CMD="${COMPRESSION_CMD:-pigz}"
+ARTIFACT_EXT="${ARTIFACT_EXT:-gz}"
+
+cd "$(dirname "${0}")"
+
+output_dir=$(dirname "${output_prefix}")
+mkdir -p "${output_dir}"
+dir="${output_dir}/tmp"
+mkdir -p "${dir}"
+
+echo "### Creating merged corpus"
+paste <(${COMPRESSION_CMD} -dc "${corpus_prefix}.${SRC}.${ARTIFACT_EXT}") <(${COMPRESSION_CMD} -dc "${corpus_prefix}.${TRG}.${ARTIFACT_EXT}") |
+  sed 's/\t/ ||| /' >"${dir}/corpus"
+
+echo "### Training alignments"
```

Comment:
Anticipating questions about this part I don't want to rewrite this in python at this point as it's mostly calling CLI commands and there will be a lot of complexity in python and it will become less readable. Also now we have a way to test the whole thing with run_task under Docker so I don't see much of a point in migration.

As for duplicating 3 lines that train alignments I also don't think extracting this will help much since it's pretty static and a standard method to train using fast_align which is unlikely to change. All other code is different across the 3 scripts.